Neos.EventSourcedNeosAdjustments/Tests/Behavior/Features/EndToEnd/Frontend/DemoSiteRenders.feature

```diff
@@ -2,7 +2,7 @@ Feature: The demo site renders
 
   Background:
     Given I start with a clean database only once per feature
-    Given I execute the flow command "site:import" with the following arguments only once per feature:
+    And I execute the flow command "site:import" with the following arguments only once per feature:
```

Comment:
I always thought `Given`, `And`, `When` and `Then` is basically ignored but no...